low-level/message_handlers/service_msg_handler.py

```diff
@@ -30,6 +32,8 @@
 from json_msgs.messages.actuators.service_controller import ServiceControllerMsg
 from json_msgs.messages.sensors.service_watchdog import ServiceWatchdogMsg
 from json_msgs.messages.actuators.ack_response import AckResponseMsg
+from framework.utils.conf_utils import (CLUSTER, GLOBAL_CONF, SRVNODE, SSPL_CONF, Conf, SITE_ID,

```

Comment:
we need to start thinking about clubbing similar constants in arrays/dictionaries, as this constants list growing slowly. No need to change in this PR
@thavanathan please around this task